core/src/main/java/jenkins/security/UpdateSiteWarningsConfiguration.java

```diff
@@ -64,6 +65,11 @@
         return Collections.unmodifiableSet(ignoredWarnings);
     }
 
+    @DataBoundSetter
+    public void setIgnoredWarnings(HashSet<String> ignoredWarnings) {
+        this.ignoredWarnings = ignoredWarnings;
```

Comment:
```suggestion
    public void setIgnoredWarnings(Set<String> ignoredWarnings) {
        this.ignoredWarnings = new HashSet<>(ignoredWarnings);
```

Would make a nicer implementation if it works with CasC.

Reply:
let me check, should be fine